co2eq/utils.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Returns the duration in hours between the start and end of an activity
+ * @param {Activity} activity Activity
+ * @returns {number} Duration in hours between start and end of an activity. Returns null if no endDatetime
+ */
+export function getActivityDurationHours(activity) {
+    if (!activity.endDatetime) return null;
+    return (activity.endDatetime - activity.datetime) / 36e5;
```

Comment:
`datetime` and `endDatetime` is JS date object, right? Is it converted to timestamps somewhere that I missed?

Reply:
yes, its a JS `date` obj - You think its necessary to use `getTime()` here?

Reply:
@madsnedergaard just pushed to use `moment()` (as its anyway already a dependency) for these calculations. Slower but certainly makes sense here!